kuma/javascript/src/article.jsx

```diff
@@ -63,6 +65,21 @@ export default function Article({ document }: DocumentProps) {
                     : 'article text-content'
             }
         >
+            {document.locale !== locale && (
+                <div id="doc-pending-fallback" class="warning">
```

Comment:
Use `className` here. Also, it feels more like an `info` message as opposed to a warning message. Will double check if we have styling for this, else I can provide you with some Sass do add.

Note to self: I am still not sure why our warning messages are red, feels like it should be yellow instead.

Reply:
Will do! Thanks for the review @schalkneethling !